gtsam/config.h.in

```diff
@@ -42,6 +42,9 @@
 // Whether we are using TBB (if TBB was found and GTSAM_WITH_TBB is enabled in CMake)
 #cmakedefine GTSAM_USE_TBB
 
+// Whether we are using a TBB version higher than 2020
+#cmakedefine TBB_GREATER_EQUAL_2020
```

Comment:
Does that mean TBB_GREATER_EQUAL_2020 was not working?

Reply:
Yes, I discovered this when trying to Ctrl-click the symbol and found out it is not there :)